changelog/updates/2024-11-19-weekly-updates.md

```diff
@@ -0,0 +1,22 @@
+- SDK: autoconf ([2.72](https://lists.gnu.org/archive/html/autotools-announce/2023-12/msg00003.html))
+- SDK: cmake ([3.30.5](https://cmake.org/cmake/help/v3.30/release/3.30.html#id5))
+- base, dev: azure-vm-utils ([0.4.0](https://github.com/Azure/azure-vm-utils/commits/v0.4.0/))
+- base, dev: bpftool ([7.4.0](https://github.com/libbpf/bpftool/releases/tag/v7.4.0))
+- base, dev: checkpolicy ([3.7](https://github.com/SELinuxProject/selinux/releases/tag/3.7))
+- base, dev: gawk ([5.3.1](https://lists.gnu.org/archive/html/info-gnu/2024-09/msg00008.html))
+- base, dev: libarchive ([3.7.7](TODO))
+- base, dev: libselinux ([3.7](https://github.com/SELinuxProject/selinux/releases/tag/3.7))
+- base, dev: libsepol ([3.7](https://github.com/SELinuxProject/selinux/releases/tag/3.7))
+- base, dev: libxml2 ([2.12.8](https://gitlab.gnome.org/GNOME/libxml2/-/releases/v2.12.8))
+- base, dev: mokutil ([0.7.2](https://github.com/lcp/mokutil/commits/0.7.2/))
+- dev: gentoolkit ([0.6.8](https://gitweb.gentoo.org/proj/gentoolkit.git/log/?h=gentoolkit-0.6.8))
+- docker: docker ([27.3.1](https://github.com/moby/moby/releases/tag/v27.3.1))
+- docker: docker-cli ([27.3.1](https://github.com/moby/moby/releases/tag/v27.3.1))
+- sysext-python: charset-normalizer ([3.4.0](https://github.com/jawah/charset_normalizer/releases/tag/3.4.0))
+- sysext-python: distlib ([0.3.9](https://github.com/pypa/distlib/blob/0.3.9/CHANGES.rst))
+- sysext-python: jaraco-functools ([4.1.0](https://github.com/jaraco/jaraco.functools/blob/v4.1.0/NEWS.rsto))
+- sysext-python: pyproject-hooks ([1.2.0](https://github.com/pypa/pyproject-hooks/blob/v1.2.0/docs/changelog.rst))
+- sysext-python: rich ([13.9.3](https://github.com/Textualize/rich/releases/tag/v13.9.3))
+- sysext-python: trove-classifiers ([2024.10.21.16](https://github.com/pypa/trove-classifiers/commits/2024.10.21.16/))
+- sysext-python: truststore ([0.10.0](https://github.com/sethmlarson/truststore/blob/v0.10.0/CHANGELOG.md))
+- vmware: libmspack ([1.11](https://github.com/kyz/libmspack/commits/v1.11/libmspack))
```

Comment:
The changelog should include something about bind 9.18.29, because bind-tools was dropped, and bind was added instead, right?